west.yml

```diff
@@ -124,7 +124,7 @@ manifest:
     - name: nrf-802154
       repo-path: sdk-nrf-802154
       path: nrf-802154
-      revision: 76308d43844018ab0734adc441464d221408cf90
+      revision: 6c8e88efe13640ef5786a70aea596755d35d3df3
```

Comment:
The `nrfxlib` PR lists the following `nrf-802154` commit: f1679be516415344873d4295dc3e9258f51a3a2c
It is different than the one used in this manifest.

Reply:
I made a mistake in nrfconnect/sdk-nrfxlib#465, I've already aligned it. The commit in this PR is correct.